File: eFormAPI/Plugins/ItemsPlanning.Pn/ItemsPlanning.Pn/Infrastructure/Models/ItemsListCasePnModel.cs
using System.Collections.Generic;

namespace ItemsPlanning.Pn.Infrastructure.Models
{
    public class ItemsListCasePnModel
    {
        public int Total { get; set; }
        public List<ItemsListPnItemCaseModel> Items { get; set; }
            = new List<ItemsListPnItemCaseModel>();
    }
}

File: eform-client/src/app/plugins/modules/items-planning-pn/models/list/items-list-case-pn.model.ts
export class ItemsListCasePnModel {
  total: number;
  items: Array<ItemsListPnItemCaseModel> = [];
}

export class ItemsListPnItemCaseModel {
  id: number;
  date: string;
  name: string;
  description: string;
  itemNumber: string;
  locationCode: string;
  buildYear: string;
  type: string;
  location: string;
  status: number;
  comment: string;
  numberOfImages: number;
  sdkCaseId: number;
}
